pkg/cmd/server/defaults.go

```diff
@@ -116,7 +117,10 @@ var defaultGRPCLogOptions = []grpclog.Option{
 		return grpclog.DefaultServerCodeToLevel(code)
 	}),
 	// changes default logging behaviour to only log finish call message
-	grpclog.WithLogOnEvents(grpclog.FinishCall),
+	grpclog.WithLogOnEvents(grpclog.PayloadReceived, grpclog.PayloadSent, grpclog.FinishCall),
```

Comment:
Since payloads may be sensitive, we should probably put this behind a flag.

I'm okay approving this and then doing a flag in a followup before the next release if you prefer, just let me know.

Reply:
I can work on the flag!